### DISCUSSION TITLE
Redirect pan gesture event to ScrollView

### DISCUSSION THREAD

**Discussion by dolsem** [software-mansion/react-native-gesture-handler#1370] [Q&A]
I have a ScrollView that takes up a portion of the screen. I am trying to respond to pan gestures from any place on the screen by scrolling the ScrollView (but make it happen automatically on the native side, without manually scrolling in JS). So in other words, I am trying to capture all pan gestures and redirect them to the ScrollView. Is it possible to accomplish this with `react-native-gesture-handler`?

**Comment by siderakis** [comment]
This would also be interesting since it would allow a PanGestureHandler (to capture touch events) to be used with a FlatList (to efficiently render a large list).

**Comment by dolsem** [comment]
My issue is actually related to #420, I'm trying to achieve similar functionality with this.
